File: LinguagensDeProgramação/Aulas/Aula26/src/aula26/Tecnico.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package aula26;

/**
 *
 * @author eport
 */
public class Tecnico extends Aluno{
    private int registroProfissional;
    
    public void praticar(){
        System.out.println("O técnico " + this.nome + " está praticando!");
    }

    public int getRegistroProfissional() {
        return registroProfissional;
    }

    public void setRegistroProfissional(int registroProfissional) {
        this.registroProfissional = registroProfissional;
    }
    
    
    
}

File: LinguagensDeProgramação/PHP/Aula41/Tecnico.php
<?php


class Tecnico extends Aluno{
    private $registroProfissional;
    
    public function praticar(){
        echo "<p>$this->nome está praticando</p>";
    }

    /**
     * Get the value of registroProfissional
     */ 
    public function getRegistroProfissional()
    {
        return $this->registroProfissional;
    }

    /**
     * Set the value of registroProfissional
     *
     * @return  self
     */ 
    public function setRegistroProfissional($registroProfissional)
    {
        $this->registroProfissional = $registroProfissional;

        return $this;
    }
}
